README.md

```diff
@@ -41,8 +51,9 @@ $ npm run start:dev
 # Visit http://localhost:3030/ in your browser
 ```
 
-<h2 align="center">Introduction</h2>
-Opportunity Calendar provides instant information on newly posted opportunuties for techies. There are times when applicants miss deadlines or get to know about available opportunties only after the deadline of their application passes. 
+## Introduction <img src="https://github.com/Samridhi-98/Images/blob/master/Images/op-about.png" width="3%">
```

Comment:
These images are not needed, and if you want to add them, they should be added in the assets folder of this project and in not your account.